cortex/blender/__init__.py

```diff
@@ -2,7 +2,13 @@
 import re
 import shlex
 import shutil
-import mda_xdrlib as xdrlib
+try:
+    import xdrlib
+except ImportError:
+    try:
+        from mda_xdrlib import xdrlib
+    except ImportError:
+        import mda_xdrlib as xdrlib 
```

Comment:
I don't think this was ever correct; so you can simplify and just use `from mda_xdrlib import xdrlib`